backend/data/src/main/kotlin/io/tolgee/service/dataImport/StoredDataImporter.kt

```diff
@@ -141,6 +146,15 @@ class StoredDataImporter(
     }
   }
 
+  private fun deleteOtherKeys() {
+    (importSettings as? SingleStepImportRequest)?.removeOtherKeys?.let {
```

Comment:
The `let` body will be executed even if the `removeOtherKeys` is `false`. Also currently I am trying to use the `let` more. It adds more structuring depth to the code and it makes it less readable. 

Instead, I would create a variable and do the check separately. 
```
val removeOtherKeys = (importSettings as? SingleStepImportRequest)?.removeOtherKeys == true
if(removeOtherKeys){
...
}
```

Reply:
Yes.